src/main/resources/lib/fotoware/xp/getConfigFromAppCfg.ts

```diff
@@ -80,13 +86,32 @@ export function getConfigFromAppCfg(): GetConfigFromAppCfgReturnType {
 				remoteAddresses[ip] = true;
 			});
 			//log.info(`remoteAddresses:${toStr(remoteAddresses)}`);
+
+			Object.keys(mappings).forEach((key) => {
+				const value = mappings[key] as string;
+				if (stringIncludes(value, ',')) {
+					mappings[key] = value.split(',');
+				}
+			});
+			// log.debug('mappings:%s', toStr(mappings));
+
 			sitesConfigs[siteName] = {
 				archiveName,
 				url,
 				remoteAddresses,
 				clientSecret,
 				clientId,
 				imports: {},
+				metadata: {
+					mappings: {
+						5: 'displayName',
+						25: 'data.tags',
+						80: 'data.artist',
+						116: 'data.copyright',
+						120: 'x.media.imageInfo.description',
```

Comment:
Can these be extracted to a separate mappings file to make it easier to extend in the future?

Reply:
Implemented